test/src/main/java/org/apache/accumulo/test/functional/SplitIT.java

```diff
@@ -92,6 +93,7 @@ protected Duration defaultTimeout() {
   @Override
   public void configureMiniCluster(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite) {
     cfg.setProperty(Property.TSERV_MAXMEM, "5K");
+    cfg.setMemory(ServerType.TABLET_SERVER, 384, MemoryUnit.MEGABYTE);
```

Comment:
Wonder if this is needed because ITs are now writing more data to the new fate table.  May be good to open a follow on issue to track down the cause of this.

Reply:
This was pretty weird, it was happening during scans according to the logs. It certainly seems related to more data being written and I agree that it should be investigated more. One option to figure it out would be to enable the JVM to dump the heap file on OOM and then we can analyze what's in the file using something like the Eclipse MAT